client/src/components/Lists/Transactions.js

```diff
@@ -156,16 +174,17 @@ export class Transactions extends Component {
 	}
 
 	searchTransactionList = async channel => {
-		let query = `from=${new Date(this.state.from).toString()}&&to=${new Date(
+		let query = this.state.from && this.state.to ? `from=${new Date(this.state.from).toString()}&&to=${new Date(
```

Comment:
Q: Why `&&` is required here?

Reply:
If values of from and to date are null, i.e, search without any date field, the new Date(null) would give Jan 1, 1970 UTC. So, we added this conditional check.

Reply:
if `from` and `to` are not null then wouldn't the query become something like this `from=10-02-2022&&to=12-02-2022` which will have `&&`.